src/env.ts

```diff
@@ -1,3 +1,31 @@
 import dotenv from 'dotenv'
 
-dotenv.config()
+import type { Env } from './types'
+
+const prepareEnv = () => {
```

Comment:
Hi, I mean `prepareEnv` function itself is needless.

Reply:
Sorry, I understand you now, haha.